### PR TITLE
Changed button text gameplay mission  model for "read, draw, write"

### PR DESCRIPTION
**Pull request by jake-linn**
# Description 
We changed the button between gameplay modes to say "Battle!" instead of "Let's go!". 
Fixes #

- What work was done? changed the text of a display button
- Why was this work done? create more engaged language for the user 
- What feature / user story is it for?- Gameplay Mission Draw, Read, & Write
- Any relevant links or images / screenshots n/a

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, tested, ready to review and merge

## Has This Been Tested

- [ ] Yes

## Checklist

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My code has been reviewed by at least one peer
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ xx] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes
- [ x] There are no merge conflicts
https://www.loom.com/share/2bfad425f4704fb79b5668f541e2d961

### CODE REVIEW

**Comment by jake-linn** [comment]
[Gameplay Mission Draw, Read, & Write - Switch button on modal to Being battle instead of let’s go](https://trello.com/c/PDUnsjlJ/738-gameplay-mission-draw-read-write-switch-button-on-modal-to-being-battle-instead-of-lets-go)

**Comment by chrislauyc** [comment]
Looks great!

**Review by xpinero — APPROVED** [review]
Whoa. I have never seen such code corrected so cleanly. Very nicely done. I really appreciate the consistency of having the first letter capitalized at the beginning of each sentence.

File: src/components/pages/GamePlay/GameDrawStep.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default function GameDrawStep(props) {
           title: 'Scribble a side quest!',
           description:
             'Grab your favorite pencil and some loose leaf sheets of paper. Based on the prompt at the end of the reading, scribble down a side quest by hand. When your story is complete, snap a photo of your pages and upload them.',
-          buttonTxt: "Let's Go!",
+          buttonTxt: 'Battle!',
```
Comment (EsdrasBrutus) [inline src/components/pages/GamePlay/GameDrawStep.js:46]: Nice simple, clean, and easy to understand changes. Looks Good

**Review by EsdrasBrutus — APPROVED** [review]

**Pull request merged**